brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
There wouldn't be a brain::neuron::morphology::Soma in any case. It would be brain::neuron::Soma even for option 2 if it's later reworked to consider brain::glia (as Stefan says, option 2 is to defer option 1 until needed and be as consistent with brion as possible for the moment).